examples/almalinux.yaml

```diff
@@ -0,0 +1,15 @@
+# This example requires Lima v0.7.0 or later.
+
+images:
+- location: "http://repo.almalinux.org/almalinux/8.5/cloud/x86_64/images/AlmaLinux-8-GenericCloud-latest.x86_64.qcow2"
+  arch: "x86_64"
+  digest: "sha512:19623e60216800fb3cd9f564c6d20415adc6d990ba4924fef2a2056035049d2ac2c749c8b71843f62af2955e8b505f4705daacb16c13d3079366b007ad846491"
+- location: "http://repo.almalinux.org/almalinux/8.5/cloud/aarch64/images/AlmaLinux-8-GenericCloud-latest.aarch64.qcow2"
+  arch: "aarch64"
+  digest: "sha512:bf8c0afaa8aaced047977de73455f09d2867907efece91cbad16dbd3ab454328b432eadccc62cdfc6e529f5147e564c9d97405a2598efde6758151c6fda758d4"
+mounts:
+- location: "~"
+- location: "/tmp/lima"
+  writable: true
+firmware:
+  legacyBIOS: true
```

Comment:
I guess we need this for Alma too?
https://github.com/lima-vm/lima/blob/f24302b6d0a8b5ea2666a6b5eea169d64c20ccea/examples/rocky.yaml#L3-L5